src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventControllerDeleteEventMappingTest.java

```diff
@@ -0,0 +1,140 @@
+package uk.gov.hmcts.darts.event.controller;
+
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import uk.gov.hmcts.darts.common.entity.EventEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.testutils.GivenBuilder;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import java.time.LocalDateTime;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.SUPER_ADMIN;
+import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.TRANSLATION_QA;
+
+@AutoConfigureMockMvc
+class EventControllerDeleteEventMappingTest extends IntegrationBase {
+
+    private static final String EVENT_MAPPINGS_ENDPOINT = "/admin/event-mappings/{event_handler_id}";
+
+    @Autowired
+    private transient MockMvc mockMvc;
+
+    @Autowired
+    private GivenBuilder given;
+
+    @Test
+    void deleteOk() throws Exception {
+        given.anAuthenticatedUserWithGlobalAccessAndRole(SUPER_ADMIN);
+        var entity = dartsDatabase.createEventHandlerData();
+        MockHttpServletRequestBuilder requestBuilder = delete(EVENT_MAPPINGS_ENDPOINT, entity.getId());
+        mockMvc.perform(requestBuilder).andExpect(status().isOk());
+    }
+
+    @Test
+    void wrongPermission() throws Exception {
+        given.anAuthenticatedUserWithGlobalAccessAndRole(TRANSLATION_QA);
+        var entity = dartsDatabase.createEventHandlerData();
+        MockHttpServletRequestBuilder requestBuilder = delete(EVENT_MAPPINGS_ENDPOINT, entity.getId());
+        MvcResult response = mockMvc.perform(requestBuilder)
+            .andExpect(status().isForbidden())
+            .andReturn();
+
+        String actualResponse = response.getResponse().getContentAsString();
+
+        String expectedResponse = """
+            {
+              "type": "AUTHORISATION_109",
+              "title": "User is not authorised for this endpoint",
+              "status": 403
+            }
+            """;
+
+        JSONAssert.assertEquals(expectedResponse, actualResponse, JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    void mappingInactive() throws Exception {
```

Comment:
Not your problem but this seems like a strange requirement. Let me check it's correct with Chris B

Reply:
So this is really just a consequence of how the frontend journey is laid out. We can only edit an active mapping, and the only way to delete a mapping is via the edit mapping screen. So as a result, we can only delete active mappings via the FE, and so here we enforce the same constraint on the BE.

There's probably a valid question on whether it's sensible to disallow deletion of inactive mappings, but that's the Figma design as it stands.

Reply:
ok, guess it can be changed if the frontend requirement ever gets changed in future.